test_bt_callback.py

```diff
@@ -0,0 +1,38 @@
+import time
```

Comment:
This is nice, thanks for creating the scripts. Could you place them in a separate folder called `examples`?

Reply:
Moved